### PR TITLE
Draft: Show error svt-av1

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -88,6 +88,9 @@ requirements:
     - eigen 3.3.*
     - jasper
     - zlib
+    # specifically compile with svt-av1 to show that
+    # they aren't compatible
+    - svt-av1  1.0
```
Comment (carterbox) [inline recipe/meta.yaml:93]:
```suggestion
    - svt-av1  1.*
```
There is no 1.0, the feedstock releases jumped from 0.9 to 1.1.

**Review by carterbox — CHANGES_REQUESTED** [review]

**Comment by carterbox** [comment]
This dependency is indirect from FFMPEG, right? There is no direct dependency of OpenCV on AV1?

File: recipe/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipe/meta.yaml]: i'm like 99% sure it is ffmpeg that uses svt-av1.

Opencv fails but it is really failing at importing the ffmpeg libraries.

We internally recompiled our ffmpeg versions for the new svt but i don't have time to fully fix things for conda-forge atm.

**Comment by jsta** [comment]
I'm not seeing this error any more. Thanks for your work on this.

**Pull request closed without merging**